Main/ClientApp/src/Components/ElementView.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Card, CardMedia } from "@material-ui/core";
+import { Theme, createStyles, makeStyles } from "@material-ui/core/styles";
+import { Element } from "../Model/Element";
+import {
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  IconButton
+} from "@material-ui/core";
+import { MoreVert } from "@material-ui/icons";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    card: {
+      background: "rgba(34, 35, 78, 0.05)"
+    },
+    link: {
+      color: "black",
+      textDecoration: "none"
+    },
+    image: {
+      display: "block",
+      margin: "auto",
+      height: "auto",
+      maxHeight: "100%",
+      width: "auto",
+      maxWidth: "100%"
+    },
+    icon: {
+      zoom: 2,
+      maxHeight: "20px",
+      maxWidth: "20px"
+    }
+  })
+);
+
+type ElementViewProps = {
+  element: Element;
+};
+
+const getDomain = (url: string) => {
```

Comment:
It works for https:// links, but what if I paste youtube.com? Please check if there is a ready library for that instead of writing our own logic.
It also doesn't belong in this component, it should be moved to a seperate file.